arrays.py

```diff
@@ -598,4 +600,8 @@
     "a Protogen",
     "a cutie",
     "a pizza",
+    "some free candy",
+    "a printer that just works",
+    "a reminder to follow this bot",
+    "a pair of programming socks",
```

Comment:
great minds think alike… #143 added “a pair of programm*er* socks” but we can certainly have both :3